### ISSUE TITLE
outdated - every now and then lists packages with identical versions

### ISSUE THREAD

**Issue by pdfkungfoo** [chocolatey/choco#843]
### Every now and then _`choco outdated`_ lists packages with identical versions.

Here is an example, happening just now, where `ack` and `Atom` are listed as outdated, with the same versions for `current` as well as `available`:

``` bash
> choco outdated                                                           
Outdated Packages                                                          
 Output is package name | current version | available version | pinned?    

ack|2.14|2.14|false                                                        
Atom|1.0.7|1.0.7|false                                                     
Far|3.0.4242|3.0.4700|false                                                

Chocolatey has determined 1 package(s) are outdated.
 2 package(s) had warnings..
Warnings:
 - ack
 - atom
```

Admittedly, there is also a warning about the same two packages. However, it is confusing: there is no explanation about what caused the so-called "warnings".
### _`choco outdated`_ should only list _really_ outdated packages...

...or it should add more details about why it thinks it should print a warning message.

**Comment by ferventcoder** [comment]
I wonder if it is #580.

**Comment by ferventcoder** [comment]
No, not the same thing. I'm guessing that will show up in the logs. Same address. I think this may be a bug?

**Comment by ferventcoder** [comment]
Typically due to not being able to find a package.

**Comment by ferventcoder** [comment]
These packages all share that they are on the community repository as well.

**Comment by brandonh-msft** [comment]
@ferventcoder 
according to the output (`|false`), no the problematic ones aren't pinned.
`clist` is still broken for online searching, it appears, but i tried to get the relevant information for ya:
```
PS C:\Users> clist "yammer-notifier" -all
Chocolatey v0.10.0 Business
0 packages found.
PS C:\Users> clist yammer-notifier -all
Chocolatey v0.10.0 Business
0 packages found.
PS C:\Users> clist yammer -lo
Chocolatey v0.10.0 Business
yammer-notifier 1.2.0.851
1 packages installed.
PS C:\Users> clist yammer -all
Chocolatey v0.10.0 Business
0 packages found.
```

I think the issue may lie w/ exempted packages after analyzing this a bit further. `yammer-notifier` has an exempted pkg on chocolatey.org, and `franz` **is** one.

**Comment by bc3tech** [comment]
@ferventcoder 
according to the output (`|false`) no, the problematic ones aren't pinned.
`clist` is still broken for online searching, it appears, but i tried to get the relevant information for ya:
```shell
PS C:\Users> clist "yammer-notifier" -all
Chocolatey v0.10.0 Business
0 packages found.
PS C:\Users> clist yammer-notifier -all
Chocolatey v0.10.0 Business
0 packages found.
PS C:\Users> clist yammer -lo
Chocolatey v0.10.0 Business
yammer-notifier 1.2.0.851
1 packages installed.
PS C:\Users> clist yammer -all
Chocolatey v0.10.0 Business
0 packages found.
```

I think the issue may lie w/ exempted packages after analyzing this a bit further. `yammer-notifier` has an exempted pkg on chocolatey.org, and `franz` **is** one.

And to answer your q in more detail:
```shell
PS C:\Users> choco pin list
Chocolatey v0.10.0 Business
notepadplusplus|6.9.2
paint.net|4.0.6
RoboForm|7.9.10.0
SourceTree|1.9.1.1
```

**Comment by ferventcoder** [comment]
I'm guessing something in the chocolatey.org codebase doesn't necessarily update the index when an exempted package is updated? So then it waits for the cache timeout to occur.

**Comment by bc3tech** [comment]
"0 packages" != "cache timeout" ;) but that's a separate issue

**Comment by ferventcoder** [comment]
Each package gets updated in the index individually and works to update caches for that package specifically.

**Comment by ferventcoder** [comment]
#351 was this same issue.

**Comment by bc3tech** [comment]
just for completeness sake i'll chime in that this is still happening in v0.10.5

**Comment by ferventcoder** [comment]
It happens when it doesn't find a package - that's when you see version old and new the same. We should add a switch here.

**Comment by ferventcoder** [comment]
We've added the switch for `--ignore-unfound` - next time you see this, run with that and see if those disappear.

**Comment by pauby** [comment]
Thanks for raising this issue. After reviewing it, it's likely been added or resolved in a more recent version. We’re closing it to keep our backlog focused. You are welcome to comment here or open a new issue if you'd like to discuss it further down the line.
